### PR TITLE
Add test for pub/sub with multi-schema

### PR DESCRIPTION
**Pull request by MarcinHoppe**

### CODE REVIEW

**Comment by MarcinHoppe** [comment]
@Particular/sqlserver-transport-maintainers I added a test that I think we were missing. Feel free to merge.

File: src/NServiceBus.SqlServer.sln.DotSettings [hunk 1]
```diff
@@ -592,6 +592,7 @@ II.2.12 &lt;HandlesEvent /&gt;&#xD;
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateBlankLinesAroundFieldToBlankLinesAroundProperty/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateThisQualifierSettings/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsCodeFormatterSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002EJavaScript_002ECodeStyle_002ESettingsUpgrade_002EJsParsFormattingSettingsUpgrader/@EntryIndexedValue">True</s:Boolean>
```
Comment (tmasternak) [inline src/NServiceBus.SqlServer.sln.DotSettings]: Should this be part of this PR?

**Comment by tmasternak** [comment]
LGTM apart from .DotSettings chagnes

**Comment by MarcinHoppe** [comment]
@tmasternak OTOH it seems overkill to update DotSettings in a standalone PR.

I can move it to a separate PR if @Particular/sqlserver-transport-maintainers deem that a better solution. No biggie since this one needs a rebase anyways.

**Comment by MarcinHoppe** [comment]
@Particular/sqlserver-transport-maintainers Please take a look again and merge.

**Pull request merged**